### PR TITLE
User guide upload

### PR DESCRIPTION
**Pull request by offset-torque**
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader.github.io/21)

### CODE REVIEW

**Comment by Frenzie** [comment]
You didn't change the homepage "User Guide" link yet. Was that on purpose?

**Comment by offset-torque** [comment]
I didn't want to modify any files outside my "user_guide" folder. After merging this PR please give me the url of the guide so I can have a final look. If I don't see any problems, you can redirect the homepage link to this one.

**Comment by Frenzie** [comment]
It looks like you didn't enable `Allow edits from maintainers` while making the PR. Could you remove the .gitkeep files?

File: user_guide/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (Frenzie) [inline user_guide/.gitkeep:1]: This one

File: user_guide/pictures/.gitkeep [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (Frenzie) [inline user_guide/pictures/.gitkeep:1]: And this one

**Comment by offset-torque** [comment]
Sorry I am still novice at github :)

**Comment by Frenzie** [comment]
It's now at http://koreader.rocks/user_guide/koreader_user_guide.html

It just occurred to me that the file should've been named index.html so that it'd work at http://koreader.rocks/user_guide without any extras.

Shall I rename it?

**Comment by offset-torque** [comment]
Yeah sure. Can you do it or should I rename or enable some setting.

**Comment by Frenzie** [comment]
Done, now straight from https://koreader.rocks/user_guide/

**Comment by offset-torque** [comment]
Thank you Frenzie. Looks good. Just one thing: "Last update" month/day is mixed up. 

Now I synced the fork, corrected the index file and ready to PR. But somehow my fork is showing "13 commits ahead" even after syncing. Shouldn't it say up to date ? Because now when I press commit, it shows 14 commits which should be only one commit I think.

**Comment by Frenzie** [comment]
You'd normally delete the branch and start a fresh one from master.

It looks like GH has an easy sync fork button there.
image

**Comment by offset-torque** [comment]
Yeah that is what I did. But I expected my commits to disappear after merging. I shouldn't still be ahead I think. Maybe I should just delete and start again as you said.

Screenshot_20231012_124756

**Questions:**
- Should I do this **"delete branch -> create new one -> edit -> PR"** flow every time I want to change something ?
- Can I delete my branch after merging, like the current one. Any extra actions or can I just delete my current branch directly.

**Comment by Frenzie** [comment]
delete branch → resync master with upstream master → create branch

For this particular the resync is presumably a bit annoying, but I don't think you can get around it.

**Pull request merged**